reprozip/native/tracer.c

```diff
@@ -231,7 +231,8 @@ int trace_add_files_from_proc(unsigned int process, pid_t tid,
     char dummy;
     char *line = NULL;
     size_t length = 0;
-    char previous_path[4096] = "";
+    size_t previous_path_size = 4096;
+    char *previous_path = malloc(previous_path_size);
```

Comment:
Since `malloc` won't initilize the memory. we need 
```
prevoius_path[0] = 0
```

Reply:
You're right, this will be fed into `strcmp()`. Fixed, thanks.